code/game/objects/structures/doors.dm

```diff
@@ -8,6 +8,8 @@
 #define DOOR_OPENING 1<<3
 ///Is currently closing
 #define DOOR_CLOSING 1<<4
+///Resin door has special properties such as ignoring intent and not being slammed shut
+#define DOOR_RESIN 1<<5
```

Comment:
Shouldn't all of these have brackets around them?

Reply:
Should they? I don't think I've seen that.

Reply:
Last I checked all bitshifts do